source/scss/typography/_component-typography.scss

```diff
@@ -0,0 +1,14 @@
+// Global component typgraphy, that does not belong to normal flowing text
+
+/* @Summary: Remove extra margin from p-tag inside a component */
+%p-tag-without-margin {
+  margin-bottom: 0;
```

Comment:
How do you think it will be duplicated? I don't understand your concern.

Reply:
%p-tag-without-margin {
  margin-bottom: 0;
}

%li-tag-without-margin {
  margin-bottom: 0;
}

%ul-tag-without-margin {
  margin-bottom: 0;
}

%foo-tag-without-margin {
  margin-bottom: 0;
}

all are doing the same

Reply:
So, one thing I can offer: Since styling also needs the possibility to configure more specific things, we can do it similar in the font-variables:

%tag-without-margin {
  margin-bottom: 0;
}

%p-tag-without-margin {
 @includes %tag-without-margin;
}

In general, there is the principle that you implement something, when you actually need it. I think, I also heard if from one of your presentations as well.

Is this something you would agree on?

Reply:
I would agree on an answer to my original question.
I question it again. Why is the "p-tag" wording so important for a generic definition.

Reply:
Answering your original question:
You "can" do it. I wouldn't recommend it, because, as said before, you need to be able to set specifics as well, independently from global definitions.

So here I am again asking you, if you would be fine with meeting in the middle.

Reply:
As I want to have the ticket moving forward after such a long time,  I take this, even that we don't have any specific case yet.
That is the same as you where mentioning, why do something that we don't have and that is not requested yet.